isis/src/base/apps/spiceinit/spiceinit.cpp

```diff
@@ -695,5 +700,9 @@ namespace Isis {
 
     delete sunPosTable;
     sunPosTable = NULL;
+
+    // Remove csminit data from the cube after successful run
+    icube->deleteGroup("CsmInfo");
```

Comment:
Does this need to be present here and on line 264-265?  It looks like they're already being deleted on success.